packages/user-operation-controller/src/helpers/PendingUserOperationTracker.test.ts

```diff
@@ -49,24 +46,20 @@ jest.mock('@metamask/controller-utils', () => ({
  * Creates a mock user operation messenger.
  * @returns The mock user operation messenger.
  */
-function createMessengerMock(): jest.Mocked<UserOperationControllerMessenger> {
+function createMessengerMock() {
   return {
     call: jest.fn(),
-    // TODO: Replace `any` with type
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  } as any;
+  } as unknown as jest.Mocked<UserOperationControllerMessenger>;
```

Comment:
Q: Should we now follow this pattern in the extension repo?

Reply:
I think it's a good general goal to try and avoid `any` so we can carry it over, excluding really awkward cases.